resources/assets/js/echo.js

```diff
@@ -0,0 +1,24 @@
+import Pusher from 'pusher-js';
+
+window.Pusher = Pusher;
+
+import Echo from 'laravel-echo';
+
+window.Echo = new Echo({
+    broadcaster: 'pusher',
+    key: import.meta.env.VITE_PUSHER_APP_KEY,
+    wsHost: import.meta.env.DEV ? '127.0.0.1' : import.meta.env.VITE_PUSHER_HOST,
```

Comment:
We can just keep this to VITE_PUSHER_HOST right? For dev environments this env variable is just set to localhost?
Besides that, does this variable even exist?

Reply:
No for dev environments it is set to the docker container name, so soketi. But that does not work for your browser. Yes it is one of the default VITE env values